### PR TITLE
Convert str concat to str interpolation

### PR DESCRIPTION
**Pull request by darkenvy**
Big commit. The console log lines are assumed safe.

By using string interpolation, we can join the "+" strings together. This makes the code appear cleaner, easier to read, and gives the reader an idea of which variables are strings at a glance.

This pull request may require more scrutiny than the others. Let me know if we should shrink this pull request.

Following AirBNB's style guide as well: http://www.jstips.co/en/javascript/keyword-var-vs-let/

### CODE REVIEW

File: lib/mwoffliner.lib.js [hunk 1]
```diff
@@ -2615,13 +2545,7 @@ function execute(argv) {
 
       var html = '<ul>\n';
       Object.keys(articleIds).sort().map(function(articleId) {
-        html =
-          html +
-          '<li><a href="' +
-          env.getArticleBase(articleId, true) +
-          '"\>' +
```
Comment (subbuss) [inline lib/mwoffliner.lib.js]: Worth checking separately (from this PR) why \> vs >?

**Comment by subbuss** [comment]
Looks like you need to resolve merge conflicts here

File: lib/mwoffliner.lib.js (hunk 1 above)
Comment (darkenvy) [reply lib/mwoffliner.lib.js]: I don't think I understand what you mean by `Worth checking separately (from this PR) why > vs >?`

But yes, I fixed the merge conflicts. I was conflicting against my own changes. All fixed up.

Comment (subbuss) [reply lib/mwoffliner.lib.js]: github ate up my characters. "\\>" vs ">" . i.e. why s the a-tag closing with a "\\>"

Comment (darkenvy) [reply lib/mwoffliner.lib.js]: That line isn't written by me originally. I'm not sure. Escaping the ">"?

**Pull request merged**